Support ntfs-3g by default without entry in /etc/filesystems for write access.

--- src/device.c
+++ src/device.c
@@ -5909,6 +5909,7 @@
   "vfat",
   "exfat",
   "ntfs",
+  "ntfs-3g",
   NULL,
 };
 